Source/WebCore/PAL/Configurations/Base.xcconfig

```diff
@@ -173,3 +166,9 @@ WK_WHICH_BUILD_SYSTEM = $(inherited)legacy
 WK_USE_NEW_BUILD_SYSTEM = $(WK_USE_NEW_BUILD_SYSTEM_$(WK_WHICH_BUILD_SYSTEM))
 WK_USE_NEW_BUILD_SYSTEM_legacy = NO
 WK_USE_NEW_BUILD_SYSTEM_not_legacy = YES
+
+WK_USE_OVERRIDE_FRAMEWORKS_DIR = $(WK_NOT_$(WK_EMPTY_$(WK_OVERRIDE_FRAMEWORKS_DIR)));
```

Comment:
Ditto about removing the definition of `WK_USE_OVERRIDE_FRAMEWORKS_DIR` from `PAL.xcconfig` since this one exists.